Yoast/wordpress-seo
# Add Bingbot presenter

## Summary


This PR can be summarized in the following changelog entry:

* Bing has added support for the same opt-in `meta robots` values as Google has, see their [announcement post](https://blogs.bing.com/webmaster/april-2020/Announcing-new-options-for-webmasters-to-control-their-snippets-at-Bing). We have decided to opt-in users of Yoast SEO by default to Bingbot crawling.

## Relevant technical choices:

* To output non-discussable robots tags, we've decided to output the value of meta robots three times: in `robots`, `googlebot` and `bingbot` tags. This leaves absolutely no possibility of confusion.
* I've moved the logic for getting to clean robots values to the Presentation layer and out of the Presenters.
* We now always concatenate these arrays with `, ` (mind the added space). Makes no difference, we did do this in the googlebot presenter, didn't go it in the robots presenter, I like consistency.

## Test instructions

This PR can be tested by following these steps:

* Go to any page that is not `noindex` you should have a `meta name bingbot` tag there, just like Googlebot.
* Go to any page that _is_ `noindex` you should *not* have a `meta name bingbot` tag there, just like no Googlebot tag.
* Other change in behavior: if a page is say `index, nofollow` you will now see that in three tags.


## UI changes

## Documentation

## Quality assurance
